cpp/include/cudf/null_mask.hpp

```diff
@@ -114,10 +114,10 @@ void set_null_mask(bitmask_type* bitmask, size_type begin_bit, size_type end_bit
  * @param stop Index of the last bit to count (exclusive)
  * @return The number of non-zero bits in the specified range
  */
-cudf::size_type count_set_bits(bitmask_type const* bitmask, size_type start, size_type stop);
+cudf::size_type count_valid_bits(bitmask_type const* bitmask, size_type start, size_type stop);
```

Comment:
Why were the names and behavior of these functions changed?  These functions don't care about null vs valid. They only care about about set vs unset bits. The null vs valid logic (and handling a null bitmask) should be layered on top of these functions.

Reply:
There's no such thing as a "null" or "valid" bit, there's only a set bit or an unset bit. We just use a bitmask that uses a set bit to represent a valid element and unset bits as a null element.